parser/epub/epub.go

```diff
@@ -53,9 +53,14 @@ func (epub *Epub) parseXML(filename string, v interface{}) error {
 	defer fd.Close()
 	dec := xml.NewDecoder(fd)
 	dec.Entity = xml.HTMLEntity
+	dec.CharsetReader = identityCharsetReader
 	return dec.Decode(v)
 }
 
+func identityCharsetReader(label string, input io.Reader) (io.Reader, error) {
+	return input, nil
```

Comment:
https://stackoverflow.com/a/50812725 - according to this comment this approach doesn't work for all encodings. Let's put TODO here with that warning